website/content/en/docs/main/tasks/private-networks/index.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Private Networks
+description: Running ClusterLink in a private network when the K8s cluster is behind a NAT or firewall.
+---
+
+
+This task involves connecting ClusterLink behind a NAT or firewall.
+To connect the ClusterLink gateway, each peer should have a public IP that will be reachable from other peers to enable cross-cluster communications. In many cases, this is not possible because clusters are behind corporate NAT or firewalls that allow outgoing connections only. For this scenario, we will use the [Fast Reverse Proxy][] (FRP) open-source project to create reverse tunnels and connect all clusters behind a private network. With FRP, only one IP needs to be public to connect all the clusters in the fabric.
+
+To create connectivity between the ClusterLink gateways, we need to set up one FRP server with a public IP and create an FRP client for each ClusterLink gateway, as illustrated below.
+
+This task includes instructions on how to connect the peers using FRP. Instructions for creating full connectivity between applications to remote services can be found in the [Nginx tutorial][] and [iPerf3 tutorial][].
+
+In this task, we will extend the peer connectivity instructions to use FRP.
+
+## Create FRP Server
+
+In this step, we will create the FRP server on the same cluster we use for ClusterLink, but it can be on any peer or Kubernetes cluster.
+
+1. Create a configmap that contains the server configuration:
+
+    ```sh
+    echo "
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+        name: frps-config
+        namespace: clusterlink-system
```

Comment:
question: might it make sense to deploy all FRP components and objects into their own `frp` namespace? It would make it clear that they're not part of CL.
The Peer CRs should change accordingly.

Reply:
I prefer to put it in  the clusterlink namespace because this is part of our deployments